chapters/ko/_toctree.yml

```diff
@@ -40,3 +40,19 @@
   sections:
   - local: events/introduction
     title: 라이브 세션과 워크샵
+
```

Comment:
Please move the unit 3 before the events section. The events section is supposed to be at the end of the TOC.

Reply:
I got it.